u8timeseries/models/arima.py

```diff
@@ -54,6 +54,10 @@ def predict(self, n):
         forecast = self.model.forecast(steps=n)[0]
         return self._build_forecast_series(forecast)
 
+    @property
+    def min_train_series_length(self) -> int:
+        return 30
```

Comment:
Why is that? Can we put docs here?

Reply:
done

Reply:
should it be 30 or 3?

Reply:
I made it 30 on purpose since ARIMA requires a significantly longer minimum length in most cases. The number is still arbitrary though.